src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx

```diff
@@ -75,22 +74,20 @@ const Profile = () => {
   const theme = useTheme();
   const { user } = useContext(UserContext);
   const [username, setUsername] = useState<string>('');
-  const [avatar, setAvatar] = useState<string>('');
-  const navigate = useNavigate();
+  // const [avatar, setAvatar] = useState<string>('');
```

Comment:
if there is commented out code, can you either remove or add //TODO or additional comments why we still need it?